iphone/Classes/TiUIWindowProxy.m

```diff
@@ -304,6 +304,7 @@ - (void)viewWillAppear:(BOOL)animated;    // Called when the view is about to ma
 - (void)viewDidAppear:(BOOL)animated;     // Called when the view has been fully transitioned onto the screen. Default does nothing
 {
 	[self updateTitleView];
+    [self updateHidesBars];
```

Comment:
tl;dr:
It's a legacy part of the code, where tabs have been used. 

The way to go here is to use the pattern that is already there. All new sources have spaces by default, but some old one have tabs. The only reason we don't run a script to fix it to spaces everywhere is, to not loose the ability to git-blame the source, because it sometimes can be very important to see why a certain change was made back in the days.